app/controllers/rents_controller.rb

```diff
@@ -1,7 +1,8 @@
 class RentsController < ApplicationController
   include Wor::Paginate
+
   def create
-    if user_signed_in? && current_user.id == params[:user_id] && request.post?
+    if user_signed_in?
       Rent.create(user_id: params[:user_id], book_id: params[:book_id],
```

Comment:
Not a correction, but you will often use current_user.id instead of params[:user_id] in cases like this. You don't want to be able to create rents for other users, probably, and if you use parmas you'll end up using stuff like the `current_user.id == params[:user_id]` you removed